webcompat/form.py

```diff
@@ -143,8 +143,10 @@
 browser_test_label = 'Did you test in another browser?'
 textarea_label = 'Please describe what happened, including any steps you took before you saw the problem'  # noqa
 
-contact_message = 'There is a mistake in the username.'  # noqa
-contact_label = 'Sharing your GitHub username—without logging in—could help us with diagnosis. This will be publicly visible.'  # noqa
+contact_message = 'There is a mistake in the username.'
+contact_label = 'Optional GitHub username'
```

Comment:
This seems to reverse a previous decision where users were thinking it was confusing. Is there additional information about it?

Reply:
Unfortunately, I don't know about the previous conversations around this text. The update comes from [the design](https://www.screencast.com/t/F0iQFCU77). As far as I know the design and the copy was approved, so I followed the design when implementing this. We may want to run this by Rosana and Christos, and see what they say, and then make a decision if we keep the copy as it is, or change it.

Reply:
For previous discussions, See 

* #1822
* #2791 
* #2762